lib/util/util.ex

```diff
@@ -204,6 +204,27 @@ defmodule Util do
 
   def local_tz, do: @local_tz
 
+  @doc """
+
+  Formats a time or a datetime so that it looks right in various
+  places on the site
```

Comment:
Can we come up with a better description? Also, not web app specific since it lives in dotcom.